config.sample.yaml

```diff
@@ -286,3 +286,8 @@ ircService:
   provisioning:
     # True to enable the provisioning HTTP endpoint. Default: true.
     enabled: true
+    # Number of seconds to wait before giving up on getting a response from
+    # an IRC operator. If the operator does not respond within the alloted
```

Comment:
Be specific when you say "operator": you mean channel operator but it's ambiguous as it stands.